packages/gatsby-source-wordpress/src/steps/create-schema-customization/helpers.js

```diff
@@ -52,14 +52,14 @@ const isWpgqlOneThirteenZeroPlus = () => {
 /**
  * This function namespaces typenames with a prefix
  */
-export const buildTypeName = name => {
+export const buildTypeName = (name, prefix) => {
   if (!name || typeof name !== `string`) {
     return null
   }
 
-  const {
-    schema: { typePrefix: prefix },
-  } = getPluginOptions()
+  if (!prefix) {
+    prefix = getPluginOptions().schema.typePrefix
+  }
```

Comment:
Allowing the prefix to be passed in rather than needing to call `getPluginOptions` means we can remove the need to use the store at runtime.